pkgs/by-name/cr/crc-reveng/package.nix

```
@@ -0,0 +1,41 @@
+{
+  stdenv,
+  fetchzip,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "crc-reveng";
+  version = "3.0.6";
+
+  src = fetchzip {
+    url = "mirror://sourceforge/reveng/${finalAttrs.version}/reveng-${finalAttrs.version}.tar.gz";
+    hash = "sha256-+MysvtpynJ8cPzAL75TlrSvUfn34BsLBjLnZt9nXIbE=";
+  };
+
+  patches = [
+    ./no_exe_extension_for_unix.patch
+    ./do_not_hardcode_gcc_in_makefile.patch
+  ] ++ (if stdenv.hostPlatform.is64bit then [ ./configure_for_64_bit_system.patch ] else [ ]);
+
+  installPhase = ''
```

```suggestion
  installPhase = ''
```
add preInstall postInstall